addon/ui/panels.py

```diff
@@ -95,14 +95,20 @@ def draw(self, context):
             col = common.split_column(box)
             col.prop(model, 'prepend_armature')
             col.prop(model, 'ignore_transforms')
+            col.prop(model, 'transform_source')
 
-            align = col.column(align=True)
-            align.prop(model, 'origin_x', text='Origin X')
-            align.prop(model, 'origin_y', text='Y')
-            align.prop(model, 'origin_z', text='-Z')
+            if model.transform_source == 'OBJECT':
+                col.prop(model, 'transform_object')
 
-            col.prop(model, 'rotation')
-            col.prop(model, 'scale')
+            else:
+                align = col.column(align=True)
+                align.prop(model, 'origin_x', text='Origin X')
+                align.prop(model, 'origin_y', text='Y')
+                align.prop(model, 'origin_z', text='-Z')
```

Comment:
The code doesn't negate Z, so should it say -Z here? I may be confused/stupid.

Reply:
Nope, my problem forgetting the - on the code i must've made a copy-paste error at some point.